spec/index.html

```diff
@@ -589,8 +589,18 @@ <h3>IRIs</h3>
       defined in RFC 3987 [[!RFC3987]].</p>
 
     <p>IRIs in the RDF abstract syntax MUST be <a data-cite="RFC3986#section-5">resolved</a>
-      per [[RFC3986]],
-      and MAY contain a fragment identifier.</p>
+      per [[RFC3986]], and MAY contain a fragment identifier. IRIs SHOULD
```

Comment:
@pchampin [appears to have been meant](https://github.com/w3c/rdf-concepts/pull/136/files#r1920505832) for `MUST NOT be relative references` to be included in the 591-593 `<p>`, and @afs [seems to have said it was added](https://github.com/w3c/rdf-concepts/pull/136/files#r1920560221), but I only see it added to the 598-602 `<p>`. Note that I've made the nouns in this text agree as plurals; they could have been made singular (and this might be better for readers) if the entire sentence were edited.
```suggestion
      per [[RFC3986]], 
      MUST NOT be relative references,
      and MAY contain fragment identifiers. IRIs SHOULD
```

Reply:
The text I see in preview is:

> IRIs in the RDF abstract syntax MUST be [resolved](https://www.rfc-editor.org/rfc/rfc3986#section-5) per [[RFC3986](https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/pull/136.html#bib-rfc3986)] and MUST NOT be [relative references](https://www.rfc-editor.org/rfc/rfc3986#section-4.2). They MAY contain a [fragment identifier](https://www.rfc-editor.org/rfc/rfc3986#section-3.5). IRIs SHOULD follow rules defined by the [IRI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1).

Reply:
Looks like this was the "git rebase" problem.